### DISCUSSION TITLE
Nuxt 3.7 won't render auto-imported component(?)

### DISCUSSION THREAD

**Discussion by felix-bohlin** [nuxt/nuxt#22880] [Questions] [answered]
### Repro 
<a href="https://stackblitz.com/edit/nuxt-starter-j94xb8?file=app.vue" target="_blank">Stackblitz</a>

### Summary:
An auto-imported component won't render even though the code appears to be valid, or am I goofing something up here?

### Context:
I have been migrating a small project from Vite to Nuxt 3 and components which have been rendered in the old environment won't show up for some reason. I don't know if there's something going on with the auto-importing but this is not the only component where I've encountered issues that weren't there before.

**Comment by xjccc** [comment, marked as answer]
` <CommentEditComment v-model="commentValue" />`

**Reply by felix-bohlin** [reply]
Wuuuuuut, did I just miss this in the docs?? Thank you!

**Reply by xjccc** [reply]
https:nuxt.com/docs/guide/directory-structure/components#component-names

you can see this
